### ISSUE TITLE
add CodePipeline and CodeBuild for Mozillians

### ISSUE THREAD

**Issue by danielhartnell** [mozilla-iam/eks-deployment#51]
I should setup CodePipeline, CodeBuild and ECR in the new account to build and push Mozillians container images. Once this is done, the Kubernetes YAML should be updated to refer to this repository.

**Comment by danielhartnell** [comment]
Duplicate of #22
